### PR TITLE
chore(docs): add Community page

### LINKED ISSUE #2109: Introduce Community page
Want to introduce a Community page to the website using the content here: https://docs.google.com/document/d/13RD3FmaABC99Nz_jHyg4CZsiXvFoIzuh85-ZhfCHeJY/edit?usp=sharing

@evwilkin did we decide where this page is going to live in the nav?

### PR DESCRIPTION
**Pull request by evwilkin**
Closes #2109 

This PR adds the new Community page.

### CODE REVIEW

**Comment by mcarrano** [comment]
This is looking great @evwilkin . So looks like you will need some graphics for this. How do you want to proceed with that? Should we open a design issue? Other ideas?

**Comment by abigaeljamie** [comment]
Thank you for building this for us, @evwilkin ! It looks great. Are you able to add a bit more padding above each header so that we have some more space between each? Similar to our Red Hat UXD site page, which has some room between each section header: https://www.redhat.com/en/about/product-design 

As for the design assets, do you think I should submit a request to the Visual Design team? I can send them this preview link so that they have the page layout handy before creating the assets.

**Comment by mcarrano** [comment]
@evwilkin @abigaeljamie I've opened a new design issue here so we can track this: https://github.com/patternfly/patternfly-design/issues/920

@abigaeljamie it would also be great if you reached out to the Visual team to see who can help with this. I can also try to get in into our next design sprint.

**Comment by abigaeljamie** [comment]
Thanks, @mcarrano ! I submitted this as a request via the [Visual Design team's request form](https://docs.google.com/forms/d/e/1FAIpQLSdsEGZB8jVKx8fCigWM-2qlj4LwFmHCHhCxPCnQ5IzpE_ziRg/viewform). I put the due date as 2 weeks from now (10/14). cc'ing @mceledonia so that he's in the loop, too!

**Comment by evwilkin** [comment]
On hold pending completion of https://github.com/patternfly/patternfly-design/issues/920

**Review by mcarrano — COMMENTED** [review]
Overall this looks great @evwilkin . But just a couple of questions/comments:

* Maybe it's my imagination but the font size on this page looks large compared with other pages. Is that possible?

* I feel like perhaps there should be more space between the image and the text block for the "Who we are" section. @mceledonia can you take a look at this? Would be interested to hear what you think.

**Comment by abigaeljamie** [comment]
Looks great, thanks @evwilkin ! Are you able to add more space between each section heading? My comment above has more detail with a linked example page.

**Comment by evwilkin** [comment]
@abigaeljamie made these changes, thanks! Updated padding above headings to 64px, the marketing site you linked has 128px padding but that looks a bit out of place to me on this page.

@mcarrano I've added 32px padding between the image & "Who we are" copy. The font size is 18px on this page which matches the Get in touch page, but throughout most of the rest of the site it is 16px. Should we update this page & the Get in touch page for consistency?

**Comment by abigaeljamie** [comment]
This is great! Thank you so much, @evwilkin !

**Comment by mcarrano** [comment]
@evwilkin I feel like 18px is too large. What do others think? I think that leaving the Get in Touch page at 18px makes sense since there is so little content on that page, but it's too large for a page with lots of content.

**Comment by evwilkin** [comment]
@mcarrano I think that's a valid point.  I've updated the font size on this page from 18px to 16px to match most other pages on the site.

**Review by mcarrano — APPROVED** [review]
Yeah, this looks much better @evwilkin . Thanks.

**Review by abigaeljamie — DISMISSED** [review]
Looks great!

File: packages/v4/patternfly-docs.routes.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ module.exports = {
     Component: () => import(/* webpackChunkName: "get-in-touch/index" */ './src/pages/get-in-touch'),
     title: 'Get in touch'
   },
+  '/community': {
+      Component: () => import('./src/pages/community'),
```
Comment (redallen) [inline packages/v4/patternfly-docs.routes.js:13]: You should add a webpack chunk name like: `/* webpackChunkName: "community/index" */ `

**Review by redallen — CHANGES_REQUESTED** [review]
Can the "Get in touch" page be moved to the last section of this new community page @mcarrano ?

**Comment by mcarrano** [comment]
@redallen I feel it's important for site users to easily find out how to contact us. Therefore I'm not in favor of moving this info to the Community page. Do you have a concern with the current placement?

**Comment by mcarrano** [comment]
@redallen after talking with @evwilkin we decided that it makes sense to go forward with this PR as is and then follow up with changes to remove the Get in Touch page in favor of adding this into the Community page and into the footer.

**Review by redallen — APPROVED** [review]
LGTM!

**Pull request merged**